koku/masu/database/trino_sql/reporting_gcpcostentrylineitem_daily_summary.sql

```diff
@@ -61,6 +74,6 @@ GROUP BY billing_account_id,
     date(usage_end_time),
     location_region,
     json_extract_scalar(json_parse(system_labels), '$["compute.googleapis.com/machine_spec"]'),
-    labels,
+    16, -- matches column num of tag's map_filter
```

Comment:
Couple questions for clarity:

1. It seems like this is being used as positional reference for tags column. Do we know if the tags column will not frequently change from being 16th? 
2. Also why positional reference vs the alias `tags` ?

Reply:
Trino doesn't like it when I reference the alias and decides to not execute the query. So I followed [what others did](https://github.com/project-koku/koku/blob/main/koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql#L310) and used the positional reference. 

There is some maintaince, but it seems to be a requirement for the query to function.

Reply:
Oh I see, thanks for the clarification.